conversion/conversion.go

```diff
@@ -550,20 +550,41 @@ func WriteSchemaFile(conv *internal.Conv, now time.Time, name string, out *os.Fi
 	// legal Cloud Spanner DDL (Cloud Spanner doesn't currently support comments).
 	// Change 'Comments' to false and 'ProtectIds' to true to write out a
 	// schema file that is legal Cloud Spanner DDL.
-	ddl := conv.SpSchema.GetDDL(ddl.Config{Comments: true, ProtectIds: false, Tables: true, ForeignKeys: true})
-	if len(ddl) == 0 {
-		ddl = []string{"\n-- Schema is empty -- no tables found\n"}
+	spddl := conv.SpSchema.GetDDL(ddl.Config{Comments: true, ProtectIds: false, Tables: true, ForeignKeys: true})
+	if len(spddl) == 0 {
+		spddl = []string{"\n-- Schema is empty -- no tables found\n"}
 	}
 	l := []string{
 		fmt.Sprintf("-- Schema generated %s\n", now.Format("2006-01-02 15:04:05")),
-		strings.Join(ddl, ";\n\n"),
+		strings.Join(spddl, ";\n\n"),
 		"\n",
 	}
 	if _, err := f.WriteString(strings.Join(l, "")); err != nil {
 		fmt.Fprintf(out, "Can't write out schema file: %v\n", err)
 		return
 	}
 	fmt.Fprintf(out, "Wrote schema to file '%s'.\n", name)
+
+	// Write another file titled {name}.ddl without comments as Spanner DDL editor UI cannot process comments
+	name = name + ".ddl"
```

Comment:
This will generate a file named '<name>.txt.ddl' - I wonder if that extension could be problematic for someone running HB on windows. How about '<name>.ddl.txt'?